### ISSUE TITLE
Requests are not properly answered

### ISSUE THREAD

**Issue by timediv** [KDD-OpenSource/fexum#166]
Requests are kept as pending and make http library return http code -1
![image](https://cloud.githubusercontent.com/assets/5778449/26685156/891a8e32-46e9-11e7-8ec4-2aa45c199b67.png)

**How to reproduce**:
- Use https://github.com/KDD-OpenSource/fexum-frontend/tree/feature/experiment-upstream
- set a target
- click on a few features
- refresh pretty soon afterwards
- Look into Chrome Network tab

**Possible problem**:
Maybe the celery queue blocks requests being answered?

**Comment by nikriek** [comment]
We should have some kind of real time queue

**Comment by timediv** [comment]
Do you know the reason why it blocks entire requests?

**Comment by nikriek** [comment]
Yes, because it's a one queue. if other tasks are still running, the "real-time" calc has to wait. The solution would be to create another queue

**Comment by timediv** [comment]
I think we shouldn't put blocking requests into queues at all.
If we do, we need some kind of guarantee that the item is quickly dealt with.

**Comment by timediv** [comment]
Sooo... how do we deal with this? @nikriek @JuliusKunze @Lichtso @marcuspappik @xasetl

**Comment by nikriek** [comment]
Still an issue?
